src/less/overwrites/_tooltip.less

```diff
@@ -3,6 +3,6 @@
   box-shadow:  0px 2px 4px 0px rgba(73, 101, 130, 0.25);
   max-width: 300px; 
   padding: 10px; 
-  font-size: 11px; 
+  font-size: 15px; 
```

Comment:
@font-size-small set by default, please introduce a new variable in the variables file in the tooltip section